_data/previous_releases.json

```diff
@@ -35,5 +36,776 @@
+      "end": "Nov 10 2021",
+      "days": 35
+    },
+    "cudf_burndown": {
+      "start": "Nov 4 2021",
+      "end": "Nov 10 2021",
+      "days": 5
+    },
+    "other_burndown": {
+      "start": "Nov 11 2021",
+      "end": "Nov 17 2021",
+      "days": 5
+    },
+    "cudf_codefreeze": {
+      "start": "Nov 11 2021",
+      "end": "Nov 17 2021",
+      "days": 5
+    },
+    "other_codefreeze": {
+      "start": "Nov 18 2021",
+      "end": "Nov 30 2021",
+      "days": 7
+    },
+    "release": {
+      "start": "Dec 8 2021",
+      "end": "Dec 9 2021",
+      "days": 2
+    }
+  },
+  {
+    "version": "21.10",
+    "cudf_dev": {
+      "start": "Jul 14 2021",
+      "end": "Sep 15 2021",
+      "days": 45
+    },
+    "other_dev": {
+      "start": "Jul 21 2021",
+      "end": "Sep 22 2021",
+      "days": 45
+    },
+    "cudf_burndown": {
+      "start": "Sep 16 2021",
+      "end": "Sep 22 2021",
+      "days": 5
+    },
+    "other_burndown": {
+      "start": "Sep 23 2021",
+      "end": "Sep 28 2021",
+      "days": 4
+    },
+    "cudf_codefreeze": {
+      "start": "Sep 23 2021",
+      "end": "Sep 28 2021",
+      "days": 4
+    },
+    "other_codefreeze": {
+      "start": "Sep 29 2021",
+      "end": "Oct 5 2021",
+      "days": 5
+    },
+    "release": {
+      "start": "Oct 6 2021",
+      "end": "Oct 7 2021",
+      "days": 2
+    }
+  },
+  {
+    "version": "21.08",
+    "cudf_dev": {
+      "start": "May 19 2021",
+      "end": "Jul 14 2021",
+      "days": 39
+    },
+    "other_dev": {
+      "start": "May 26 2021",
+      "end": "Jul 21 2021",
+      "days": 39
+    },
+    "cudf_burndown": {
+      "start": "Jul 15 2021",
+      "end": "Jul 21 2021",
+      "days": 5
+    },
+    "other_burndown": {
+      "start": "Jul 22 2021",
+      "end": "Jul 28 2021",
+      "days": 5
+    },
+    "cudf_codefreeze": {
+      "start": "Jul 22 2021",
+      "end": "Jul 28 2021",
+      "days": 5
+    },
+    "other_codefreeze": {
+      "start": "Jul 29 2021",
+      "end": "Aug 3 2021",
+      "days": 4
+    },
+    "release": {
+      "start": "Aug 4 2021",
+      "end": "Aug 5 2021",
+      "days": 2
+    }
+  },
+  {
+    "version": "21.06",
+    "cudf_dev": {
+      "start": "Mar 25 2021",
+      "end": "May 18 2021",
+      "days": 39
+    },
+    "other_dev": {
+      "start": "Apr 1 2021",
+      "end": "May 25 2021",
+      "days": 39
+    },
+    "cudf_burndown": {
+      "start": "May 19 2021",
+      "end": "May 25 2021",
+      "days": 5
+    },
+    "other_burndown": {
+      "start": "May 26 2021",
+      "end": "Jun 2 2021",
+      "days": 5
+    },
+    "cudf_codefreeze": {
+      "start": "May 26 2021",
+      "end": "Jun 2 2021",
+      "days": 4
+    },
+    "other_codefreeze": {
+      "start": "Jun 3 2021",
+      "end": "Jun 8 2021",
+      "days": 4
+    },
+    "release": {
+      "start": "Jun 9 2021",
+      "end": "Jun 10 2021",
+      "days": 2
+    }
+  },
+  {
+    "version": "0.19",
```

Comment:
The rending of 0.19 and 0.18 are broken: 
`{“start”=>”Apr 21 2021”, “end”=>”Apr 22 2021”, “days”=>2}`



`

Reply:
Fixed!